po/es.po

```diff
@@ -1058,15 +1058,14 @@ msgid ""
 "  -l, --lock-secs SEC           after failed login lock account for SEC "
 "seconds\n"
 msgstr ""
-"  -l, --lock-secs SEG           después de fallar al acceder la cuenta se\n"
-"                                bloqueará después de SEG segundos\n"
+"  -l, --lock-secs SEG           después de fallar al acceder a la cuenta\n"
+"                                se bloqueará después de SEG segundos\n"
```

Comment:
IIUC from the English version, if a failed login happens then the account will be locked for SEC seconds. In my opinion, as a Spaniard, is that the second `después` doesn't make sense with the aforementioned understanding. Although I admit that it may make sense for our Latin Americans colleagues. I would suggest to change the phrase to: `después de fallar al acceder a la cuenta esta se bloqueará durante/por SEG segundos`. We should choose between `durante` or `por`. By the way, I'm open to discussion as my previous suggestion could be incorrect for other dialects.

Reply:
I had not read properly the English version (just fixed the typo), but you're absolutely right. It's the same meaning for Latin America (argentinean here). I think `por` is good enough.